source/_patterns/00-atoms/components/speech-bubble.yaml

```diff
@@ -0,0 +1,37 @@
+assets:
+  css:
+    - speech-bubble.css
+    - ../../../../assets/sass/patterns/molecules/hypothesis-opener.css
+  js: []
+schema:
+  $schema: http://json-schema.org/draft-04/schema#
+  type: object
+  properties:
+    text:
+      type: string
+      minLength: 1
+    isSmall:
+      type: boolean
+      enum:
+        - true
+    hasPlaceholder:
+      type: boolean
+      enum:
+        - true
+  required:
+    - text
+  anyOf:
```

Comment:
Any reason why you couldn't have both (even though we don't use it now)? Or doesn't it work?

Reply:
No reason it shouldn't exist, but I haven't built the css for that as it's not required at the moment.